exporter/src/main/java/com/lightstep/opentelemetry/exporter/Adapter.java

```diff
@@ -284,4 +294,33 @@ static long spanIdToLong(final SpanId spanId) {
   private static long fromByteArray(byte[] bytes) {
     return ByteBuffer.wrap(bytes).getLong();
   }
+
+  /**
+   * Determines if a SpanData is an errored Span, filtering out 4XX server responses as defined by
+   * https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/http.md#status
+   * @param spanData the span under question
+   * @return whether or not the span is an error
+   */
+  static boolean isError(SpanData spanData) {
+    if (spanData.getStatus().isOk()) {
+      return false;
+    }
+    // status is not ok and there is no special case for being a server
+    if (spanData.getKind() != Kind.SERVER) {
+      return true;
+    }
+
+    // special case for servers, 4XX responses are not errors
+    switch (spanData.getStatus().getCanonicalCode()) {
```

Comment:
Can you change this to use `spanData.getStatus().isOK()` to check for the correct status?   I realize that will undo the 4xx change that you wanted but we'd like to maintain consistency with a few other places that treat all non-zero statuses as errors, including the Jaeger exporter for OTel (https://github.com/open-telemetry/opentelemetry-java/blob/master/exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/Adapter.java#L116-L117) and an upcoming Lightstep satellite release.  The underlying reason that other places do this is that it's difficult to say that, for *all* applications, 4xx is not an error that someone cares about.

There's some on-going discussion in the OTel community about allowing users to explicitly mark which statuses or exceptions are viewed as "really an error - something bad happened", so this would be explicitly configurable as part of OTel if that work goes through.

Reply:
Since there is precedence in the jaeger adapter I'll put it back how it was, but I would like to see some configurability in the future.

Reply:
I've pushed the commit to my fork, please have another look.